SteakBot.Core/Modules/MemeModule.cs

```diff
@@ -42,10 +42,60 @@ public async Task AddMeme(string name, string value, string description)
             }
             else
             {
-                replyMessage = "Saving failed! ;(";
+                replyMessage = "Add failed! ;(";
             }
 
             await ReplyAsync(replyMessage, embed: embed?.Build());
         }
+
+        [Command("editMeme")]
+        public async Task EditMeme(string name, string newName, string newValue, string newDescription)
+        {
+            var replyMessage = string.Empty;
+            EmbedBuilder embed = null;
+
+            var type = _memeService.GetMemeType(newValue);
+
+            var result = _memeService.EditCommand(name, new MemeCommand(type, newName, newValue, newDescription));
+
+            if (result)
+            {
+                if (type == MemeResultType.Image)
+                {
+                    embed = new EmbedBuilder
+                    {
+                        ImageUrl = newValue
+                    };
+                }
+                else
+                {
+                    replyMessage = newValue;
+                }
+            }
+            else
+            {
+                replyMessage = "Edit failed! ;(";
+            }
+
+            await ReplyAsync(replyMessage, embed: embed?.Build());
+        }
+
+        [Command("removeMeme")]
+        public async Task RemoveMeme(string name)
+        {
+            var result = _memeService.RemoveCommand(name);
+
+            var replyMessage = string.Empty;
+            if (result)
+            {
+                replyMessage = "Removed";
+            }
+            else
+            {
+                replyMessage = "Remove failed! ;(";
+            }
+
+            await ReplyAsync(replyMessage);
```

Comment:
This can be condensed to a single line using the ternary operator.

Reply:
In fact the whole method body can be condensed to 1 line, or 2 at most.

Reply:
I will convert the if to use the ternary operator but I am not condensing everything into a single line!